templates/kubernetes/docs/shared/_side-navigation.md

```diff
@@ -1,92 +1,32 @@
-## About
-- [Home](/kubernetes/docs)
-- [Overview](/kubernetes/docs/overview)
+- [Home](/kubernetes/docs/)
 
-## Supported versions
-- [1.28](/kubernetes/docs/1.28/components)
-- [1.27](/kubernetes/docs/1.27/components)
-- [1.26](/kubernetes/docs/1.26/components)
-- [Version info](/kubernetes/docs/supported-versions)
-
-## Install
-- [Quickstart](/kubernetes/docs/quickstart)
-- [Install](/kubernetes/docs/install-manual)
-- [Local install](/kubernetes/docs/install-local)
-- [Offline install](/kubernetes/docs/install-offline)
-- [NVIDIA DGX](/kubernetes/docs/nvidia-dgx)
-
-## Cloud Integration
-- [AWS integration](/kubernetes/docs/aws-integration)
-- [Azure integration](/kubernetes/docs/azure-integration)
-- [GCP integration](/kubernetes/docs/gcp-integration)
-- [OpenStack integration](/kubernetes/docs/openstack-integration)
-- [vSphere integration](/kubernetes/docs/vsphere-integration)
-- [LXD](/kubernetes/docs/install-local)
-- [Equinix Metal](/kubernetes/docs/equinix)
+## Tutorials
+- [Get started](/kubernetes/docs/quickstart)
 
-## Networking
-- [CNI overview](/kubernetes/docs/cni-overview)
-- [Flannel](/kubernetes/docs/cni-flannel)
-- [Calico](/kubernetes/docs/cni-calico)
-- [Canal](/kubernetes/docs/cni-canal)
-- [Cilium](/kubernetes/docs/cni-cilium)
-- [Kube OVN](/kubernetes/docs/cni-kube-ovn)
-- [Tigera Secure EE](/kubernetes/docs/tigera-secure-ee)
-- [Multus](/kubernetes/docs/cni-multus)
-- [SR-IOV](/kubernetes/docs/cni-sriov)
-- [Using multiple networks](/kubernetes/docs/multiple-networks)
-- [IPv6](/kubernetes/docs/ipv6)
-- [Ingress](/kubernetes/docs/ingress)
+## How to...
 
-## Container Runtimes
-- [Containerd & Docker](/kubernetes/docs/container-runtime)
-- [Kata](/kubernetes/docs/kata)
-- [VM workloads (KubeVirt)](/kubernetes/docs/kubevirt)
+- [...install](/kubernetes/docs/how-to-install)
+- [...integrate with clouds](/kubernetes/docs/how-to-clouds)
+- [...configure CNI](/kubernetes/docs/how-to-cni)
+- [...use container runtimes](/kubernetes/docs/how-to-containers)
+- [...operate Kubernetes](/kubernetes/docs/how-to-operate)
+- [...achieve High Availability](/kubernetes/docs/how-to-ha)
+- [...secure your cluster](/kubernetes/docs/how-to-security)
```

Comment:
Are you sure you want to do navigation like this? It's rather difficult to read or scan. Also, it's going to be very annoying for screenreaders. You could use `&hellip;` but either way, I would suggest real words are better.